vignettes/annotations.Rmd

```diff
@@ -107,13 +107,32 @@ function(name, age) {
 #* @serializer rds
 #* @parser multi
 #* @parser rds
-#* @param f:file
+#* @param f:file A file
 #* Upload an rds file and return the object
 function(f) {
   as_attachment(f[[1]], names(f)[1])
 }
 ```
 
+##### Equivalent programmatic usage
+```{r, eval = FALSE, echo = TRUE}
+f <- function(name, age) { sprintf("%s is %i years old", name, age) }
+g <- function(f) { as_attachment(f[[1]], names(f)[1]) }
```

Comment:
These programmatic usage sections are AWESOME! Whenever they can be added (or makes sense to add), I'm all for it!